proposals/0107-unsaferawpointer.md

```diff
@@ -1477,34 +1477,36 @@ struct UnsafeMutableRawPointer : Strideable, Hashable, _Pointer {
   func bindMemory<T>(to: T.Type, capacity: Int) -> UnsafeMutablePointer<T>
   func assumingMemoryBound<T>(to: T.Type) -> UnsafeMutablePointer<T>
 
-  func initialize<T>(_: T.Type, with: T, count: Int = 1)
+  func initialize<T>(_: T.Type, to: T, count: Int = 1)
     -> UnsafeMutablePointer<T>
-  func initialize<T>(toContiguous: T.Type, atIndex: Int, with: T)
+  func initialize<T>(contiguous: T.Type, at: Int, to: T)
```

Comment:
I think the "as" label makes sense when working with a raw pointer.

I like this:

func initialize<T>(as: T.Type, at: Int = 0, count: Int = 1, to value: T) -> UnsafeMutablePointer<T>

address.initialize(as: String.self, to: "hello")
address.initialize(as: String.self, at: 3, to: "hello")
address.initialize(as: String.self, count: 3, to: "hello")
address.initialize(as: String.self, at: 3, count: 3, to: "hello")